src/Uno.UWP/UI/Input/GestureRecognizer.cs

```diff
@@ -179,8 +179,8 @@ private void Recognize(List<PointerPoint> points, PointerPoint pointerUp)
 
 		#region Tap (includes DoubleTap)
 		internal const ulong MultiTapMaxDelayTicks = TimeSpan.TicksPerMillisecond * 1000;
-		internal const int TapMaxXDelta = 15;
-		internal const int TapMaxYDelta = 15;
+		internal const int TapMaxXDelta = 10;
```

Comment:
Why ? (why previous, why new)

Reply:
Previous : Random
New: Still random ... but found that the delta was a bit too high to use it properly in the `ToggleSwitch`. I've did a commit just for this change in order to be able to track it a bit more easily : https://github.com/unoplatform/uno/pull/2249/commits/93c6b674afa2b1ed69212ec804711558774c26f3